carta/session.py

```diff
@@ -417,6 +417,43 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(Boolean())
+    def set_viewer_mode(self, multiple=True):
+        """
+        Switch between single-panel mode and multiple-panel mode.
+
+        Parameters
+        ----------
+        multiple : {0}
+            Whether to use multiple-panel mode. By default is ``True``.
+        """
+        self.call_action("widgetsStore.setImageMultiPanelEnabled", multiple)
+
+    def previous_page(self):
+        """Go to previous page on viewer"""
+        self.call_action("widgetsStore.onPreviousPageClick")
+
+    def next_page(self):
+        """Go to next page on viewer"""
+        self.call_action("widgetsStore.onNextPageClick")
+
+    @validate(Number(), Number())
+    def set_viewer_grid(self, n_columns=2, n_rows=2):
+        """
+        Set numbers of columns and rows for viewer grids.
```

Comment:
I suggest `Set number of columns and rows in viewer grid.`

Reply:
Done in d44b668.